hs_core/views/utils.py

```diff
@@ -695,3 +716,11 @@ def move_or_rename_file_or_folder(user, res_id, src_path, tgt_path):
         rename_irods_file_or_folder_in_django(resource, src_full_path, tgt_full_path)
 
     hydroshare.utils.resource_modified(resource, user)
+
+
+def irods_path_is_allowed(path):
+    """ paths containing '/../' are suspicious """
+    if path == "":
+        raise ValidationError("Empty file paths are not allowed")
+    if '/../' in path:
+        raise SuspiciousFileOperation("File paths cannot contain '/../'")
```

Comment:
@alvacouch I was thinking about the case where ```./A/B``` is used. I just tested using icommands ```ils ./<resource_short_id>``` and it lists the collection/directory for this specific resource. So iRODS icommand treats ```.``` as the home directory. So probably it is better to prevent ```.``` in the path to protect from possible hacker attack.